src/routes.py

```diff
@@ -22,16 +24,14 @@ def userpage():
 
 @app.route("/login", methods=["GET","POST"])
 def login():
-    if request.method == "GET":
-        return render_template("login.html")
     if request.method == "POST":
         username = request.form["username"]
         password = request.form["password"]
         if user_service.login(username,password):
             return redirect("/")
-        else:
-            flash(f"Login failed")
-            return redirect("/")
+        flash("Login failed")
+        return redirect("/")
```

Comment:
I dislike this specific lint (`no-else-return`), because it goes against common practices and does not express intent very well. Either way, no big deal in this project.

Reply:
I think you are right that with else the intent can be expressed better